tests/integration/targets/vmware_content_template/tasks/main.yml

```diff
@@ -60,6 +60,20 @@
           - __res.template_info.msg == "Template '" + template_name + "'."
       when: not run_on_simulator
 
+    - name: Delete template from content library
+      vmware.vmware.content_template:
+        validate_certs: false
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        port: "{{ vcenter_port }}"
+        template: "{{ template_name }}"
+        library: "{{ library }}"
+        vm_name: "{{ vm }}"
```

Comment:
vm_name is a mandatory variable for 'absent' state? It seems wrong to me.

Reply:
created a ticket in jira for the next release :+1: